components/eam/src/chemistry/mozart/lin_strat_chem.F90

```diff
@@ -273,7 +278,7 @@ subroutine lin_strat_chem_solve( ncol, lchnk, o3col, temp, sza, pmid, delta_t, r
     call outfld( 'LINOZ_O3COL'  , o3col_du_diag          , ncol, lchnk )
```

Comment:
Can you add the variable in `outfld` to python files.

Reply:
Yes, this can be done.  Note though that it is a vertically-integrated quantity, so you will not be able to reproduce it using inputs at just a single vertical level, as I had been doing for these subroutines.  Do you want me to go ahead and write out inputs / outputs for all vertical levels in a column as well?

Reply:
Yes.

Reply:
We could add vertically-integrated quantity only in tests with multiple vertical columns.